docs/getting-started/install-elastic-endpoint.asciidoc

```diff
@@ -81,6 +81,7 @@ If the endpoint is running {stack} version 7.17.0 or earlier:
 --
 image::images/fda/fda-7-16.png[]
 
+// We might need to remove this section in a separate PR since this PR contains content that can only be backported to 8.6 or 8.7.
```

Comment:
Maybe create a separate issue to get this in our backlog, instead of a comment that could more easily get overlooked or forgotten

Reply:
Np, task is tracked here: https://github.com/elastic/security-docs/issues/3477

Reply:
I'll remove the comment as well since it's covered in 3477.

```suggestion
```